docs/Developer-Guide.md

```diff
@@ -228,16 +229,70 @@ $ go get -d -u github.com/kata-containers/kata-containers
 $ cd $GOPATH/src/github.com/kata-containers/kata-containers/src/agent && make
 ```
 
-The agent is built with seccomp capability by default.
-If you want to build the agent without the seccomp capability, you need to run `make` with `SECCOMP=no` as follows.
+### Build the agent with seccomp
 
+The agent is built with seccomp feature by default.
+You **MUST** install the `libseccomp` library in your build environment and need to set the environment
+variables for the [`libseccomp` crate](https://github.com/libseccomp-rs/libseccomp-rs) as necessary to
+link the agent binary with a static `libseccomp` library.
+
+#### Link the `libseccomp` library statically against `musl`
+
+Install a static `libseccomp` library for `musl`:
+
+```sh
+$ sudo $GOPATH/src/github.com/kata-containers/kata-containers/ci/install_libseccomp.sh /tmp/kata-libseccomp /tmp/kata-gperf
```

Comment:
Suggest changing the installation path to a directory that is not under `/tmp`, they will disappear after reboot.

Reply:
I use `/usr/local/kata-libseccomp` and `/usr/local/kata-gperf` instead of `/tmp`.